hal/src/sercom/spi/async_api/dma.rs

```diff
@@ -6,7 +6,7 @@ use crate::{
     dmac::{
         channel::{self, Channel},
         sram::DmacDescriptor,
-        AnyChannel, Beat, Buffer, ReadyFuture,
+        AnyChannel, Beat, Buffer, ReadyChannel, ReadyFuture,
```

Comment:
My mistake, I just realized that this is a bug - can you remove the `ReadyChannel` import from this file, and replace all occurences of `ReadyChannel` with `ReadyFuture`?